packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -847,6 +850,16 @@ describe('#integration - StripeHelper', () => {
   });
 
   describe('createSubscriptionWithPMI', () => {
+    it('checks that roundTime() returns time rounded to the nearest minute', async () => {
```

Comment:
Test looks good. Could we move it to its own test file `test/local/payments/utils`? (This probably should have happened when the generateIdemptotencyKey function was created. That was my bad)